sprout/Dockerfile

```diff
@@ -4,9 +4,9 @@ MAINTAINER maintainers@projectclearwater.org
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes sprout
 RUN sed -e 's/\(echo 0 > \/proc\/sys\/kernel\/yama\/ptrace_scope\)/# \0/g' -i /etc/init.d/sprout
 
-COPY chronos.supervisord.conf /etc/supervisor/conf.d/chronos.conf
-COPY memcached.supervisord.conf /etc/supervisor/conf.d/memcached.conf
 COPY sprout.supervisord.conf /etc/supervisor/conf.d/sprout.conf
 COPY clearwater-group.supervisord.conf /etc/supervisor/conf.d/clearwater-group.conf
+COPY plugins/* /usr/share/clearwater/clearwater-cluster-manager/plugins/
```

Comment:
Arguably this isn't only needed in Docker - it's needed anywhere we split memcached out but don't use Astaire as a proxy - and so it might be worth putting these in a separate package (so they can be deployed in e.g. OpenStack if we want to demo this topology there). As we're merging the Astaire stuff in a few weeks anyway, that's probably not worth doing.